akka-actor-testkit-typed/src/main/scala/akka/actor/testkit/typed/internal/TestProbeImpl.scala

```diff
@@ -244,51 +249,55 @@ private[akka] final class TestProbeImpl[M](name: String, system: ActorSystem[_])
 
   private def receiveMessages_internal(n: Int, max: FiniteDuration): immutable.Seq[M] = {
     val stop = max + now
-    for (x ← 1 to n) yield {
+    for (x <- 1 to n) yield {
       val timeout = stop - now
       val o = receiveOne_internal(timeout)
       o match {
-        case Some(m) ⇒ m
-        case None    ⇒ assertFail(s"timeout ($max) while expecting $n messages (got ${x - 1})")
+        case Some(m) => m
+        case None    => assertFail(s"timeout ($max) while expecting $n messages (got ${x - 1})")
       }
     }
   }
 
-  override def fishForMessage(max: FiniteDuration, hint: String)(fisher: M ⇒ FishingOutcome): immutable.Seq[M] =
+  override def fishForMessage(max: FiniteDuration, hint: String)(fisher: M => FishingOutcome): immutable.Seq[M] =
     fishForMessage_internal(max.dilated, hint, fisher)
 
-  override def fishForMessage(max: FiniteDuration)(fisher: M ⇒ FishingOutcome): immutable.Seq[M] =
+  override def fishForMessage(max: FiniteDuration)(fisher: M => FishingOutcome): immutable.Seq[M] =
     fishForMessage(max, "")(fisher)
 
   override def fishForMessage(max: JDuration, fisher: java.util.function.Function[M, FishingOutcome]): JList[M] =
     fishForMessage(max, "", fisher)
 
-  override def fishForMessage(max: JDuration, hint: String, fisher: java.util.function.Function[M, FishingOutcome]): JList[M] =
+  override def fishForMessage(max: JDuration,
+                              hint: String,
+                              fisher: java.util.function.Function[M, FishingOutcome]): JList[M] =
     fishForMessage_internal(max.asScala.dilated, hint, fisher.apply).asJava
 
-  private def fishForMessage_internal(max: FiniteDuration, hint: String, fisher: M ⇒ FishingOutcome): List[M] = {
+  private def fishForMessage_internal(max: FiniteDuration, hint: String, fisher: M => FishingOutcome): List[M] = {
     @tailrec def loop(timeout: FiniteDuration, seen: List[M]): List[M] = {
       val start = System.nanoTime()
       val maybeMsg = receiveOne_internal(timeout)
       maybeMsg match {
-        case Some(message) ⇒
-          val outcome = try fisher(message) catch {
-            case ex: MatchError ⇒ throw new AssertionError(
-              s"Unexpected message $message while fishing for messages, " +
-                s"seen messages ${seen.reverse}, hint: $hint", ex)
+        case Some(message) =>
+          val outcome = try fisher(message)
+          catch {
```

Comment:
Weird indenting again. Can we have

```scala
val outcome =
  try fisher(message)
  catch {
  }
```

?

Reply:
+1 that `try catch` line and indent change is bad

Reply:
I agree, but how?

Reply:
fixed this manually before formatting